primitiv/_function.pyx

```diff
@@ -205,6 +205,20 @@ class functions:
     def stop_gradient(Node x):
         return wrapNode(func_stop_gradient(x.wrapped))
 
+    @staticmethod
+    def conv2d(Node x, Node w, unsigned padding0, unsigned padding1, unsigned stride0, unsigned stride1, unsigned dilation0, unsigned dilation1):
```

Comment:
These lines look too long to show and I recommends to add some newlines appropriately similar to the next line.

Reply:
Fixed.